src/guide/essentials/conditional.md

```diff
@@ -15,15 +15,15 @@ const awesome = ref(true)
 
 ## `v-if`
 
-The directive `v-if` is used to conditionally render a block. The block will only be rendered if the directive's expression returns a truthy value.
+`v-if` ko'rsatuvchisi (directive) blok kodni shartli ko'rsatish uchun ishlatiladi. Blok kod `v-if` ko'rsatuvchisining ifodasi(expression) rost qiymat qaytargandagina sahifada ko'rinadi.
```

Comment:
![image](https://user-images.githubusercontent.com/12369295/172680680-b0e56c00-08a1-4170-a599-11b0f91d8a16.png)
ifodasidan keyin probel bo'lishi kerak.

Reply:
To'g'rilandi.